src/ralph/models/converter.py

```diff
@@ -24,18 +21,50 @@ class ConversionItem:
     transformers: tuple[Callable[[Any], Any]]
 
     def __init__(self, dest: str, src=None, transformers=lambda _: _):
+        """Initializes ConversionItem.
+
+        Args:
+            dest (str): The destination path where to place the converted value.
+            src (str or None): The source from where the value to convert is fetched.
+                - When `src` is a path (ex. `context__user_id`) - the value is the item
+                    of the event at the path.
+                - When `src` is `*` - the value is the whole event.
+                - When `src` is `**` - the value is the original event string.
+                - When `src` is `None` - the value is `None`.
```

Comment:
I would suggest to add a `raw_input` boolean argument instead (which defaults to `False`) and add a `raw_input` attribute that will be used in the `get_value` method (see other comments). WDYT?

Reply:
Good Idea! Thanks) Updated this part to reduce the special cases:
Now, when `src` is None the whole event is passed else it's a path and we fetch a specific field.